package.json

```diff
@@ -93,7 +93,7 @@
   ],
   "engines": {
     "node": ">=10.16.3 <10.17.0",
-    "yarn": ">=1.17.3 <1.18.0"
+    "yarn": ">=1.17.3 <1.18.0 || 1.19.1"
```

Comment:
Oops I forgot to remove this from the PR, but we should file an issue to make LWC work with yarn 1.19.1 (which includes updating all the volta stuff)